### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by UmarMaishanu** [Cyfrin/foundry-full-course-cu#2214] [Q&A]
Please casn someone tell me why my compiler isn’t compiling unlike before?

**Comment by Maa-ly** [comment]
Could you please provide more specific details about the problem? It would be helpful to know.
Are there any specific error messages or warnings you're encountering? If so, please share them.

**Comment by EngrPips** [comment]
Hello @UmarMaishanu, Please visit  https://github.com/Cyfrin/foundry-full-course-cu/blob/main/how-to-ask-a-question.md so that you can be informed on how to ask an appropriate question.

**Comment by UmarMaishanu** [comment]
Here is the error shown whenever I try to compile “
Worker error: Uncaught NetworkError: Failed to execute 'importScripts' on
'WorkerGlobalScope': The script at '
https://remix.ethereum.org/assets/js/soljson.js' failed to load.

**Reply by EngrPips** [reply]
Please refresh your browser and retry.

**Comment by UmarMaishanu** [comment]
It worked. Thank you.

**Reply by EngrPips** [reply]
You are welcome.
